src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -2889,6 +2889,11 @@ theorem msb_rotateLeft {m w : Nat} {x : BitVec w} :
   · simp
     omega
 
+@[simp]
+theorem toNat_rotateLeft {x : BitVec w} {r : Nat} :
+    (x.rotateLeft r).toNat = (x.toNat <<< (r % w)) % (2^w) ||| x.toNat >>> (w - r % w) := by
+  simp only [rotateLeft, rotateLeftAux, toNat_shiftLeft, toNat_ushiftRight, toNat_or]
```

Comment:
Could we please have a lemma e.g. `rotateLeft_def` which combines the first two steps here? Similarly for right.